axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveTemplateService.java

```diff
@@ -286,25 +288,29 @@ public List<Long> generateMove(LocalDate moveDate, List<HashMap<String, Object>>
                   .mapToInt(Integer::parseInt)
                   .toArray();
         }
+        moveTemplatePartner = fillPartnerWithMoveTemplate(moveTemplate);
 
         BankDetails companyBankDetails = null;
         if (moveTemplate != null
             && moveTemplate.getJournal() != null
             && moveTemplate.getJournal().getCompany() != null) {
           companyBankDetails =
               bankDetailsService.getDefaultCompanyBankDetails(
-                  moveTemplate.getJournal().getCompany(), null, null, null);
+                  moveTemplate.getJournal().getCompany(),
+                  moveTemplatePartner.getOutPaymentMode(),
```

Comment:
this will throw a NPE since moveTemplatePartner can be null

Reply:
Fixed